packages/xo-web/src/common/xo/copy-vms-modal/index.js

```diff
@@ -59,9 +72,41 @@ class CopyVmsModalBody extends BaseComponent {
   _onChangeNamePattern = event =>
     this.setState({ namePattern: event.target.value })
 
+  _getVmsWithNoZstdCompression = createSelector(
+    () => this.props.vms,
+    () => this.props.containers,
+    createCollectionWrapper((vms, containers) => {
+      const vmIds = []
+      for (const id in vms) {
+        const container = containers[vms[id].$container]
+        if (!container.zstdSupported) {
+          vmIds.push(id)
+        }
+      }
+      return vmIds
+    })
+  )
+
+  _getVmsWithNoZstdCompressionLink = createSelector(
+    this._getVmsWithNoZstdCompression,
+    vms => ({
+      pathname: '/home',
+      query: {
+        t: 'VM',
+        s: constructQueryString({
+          id: {
+            __or: vms,
+          },
+        }),
+      },
+    })
+  )
+
   render() {
     const { formatMessage } = this.props.intl
     const { compression, namePattern, sr } = this.state
+    const nVms =
+      compression === 'zstd' ? this._getVmsWithNoZstdCompression().length : 0
```

Comment:
It would make more sense if `compression === 'zstd'` was part of the condition to show the error message.

Reply:
I can inline it. I created a const to call the selector one time but it's not a big deal.

Reply:
But when you read the JSX, you'd like to see:
```js
compression === 'zstd' && nVmsWithoutZstd > 0 &&
  <Tooltip ... />
```
It helps a lot understand what you're actually doing because you can easily read it in natural language. And you don't need to inline it if you don't want to. (But calling a selector multiple times in a row is not an issue either and is kind of the whole point of selectors :) )